dnf/cli/cli.py

```diff
@@ -803,6 +803,11 @@ def configure(self, args, option_parser=None):
             e = '%s: %s' % (ucd(e.args[1]), repr(e.filename))
             logger.critical(_('Config error: %s'), e)
             sys.exit(1)
+        if opts.destdir is not None:
+            self.base.conf.destdir = opts.destdir
+            if not self.base.conf.downloadonly and opts.command != 'download':
```

Comment:
Regarding line too long: I think that splitting line 809 would be counterproductive and would bring no real benefit except conforming to the rules set by the linter just because they exist.

PEP8 sets the limit to 79, probably because terminals default to 80 characters per line. I do not know why DNF has set the max line length to 100, however.

I have fixed line 1072.